pkg/operator/status.go

```diff
@@ -87,7 +87,12 @@ func (c *Controller) syncStatus(cr *imageregistryv1.Config, deploy *appsapi.Depl
 		Message: "",
 		Reason:  "",
 	}
-	if deploy == nil {
+
+	if removed {
```

Comment:
https://github.com/openshift/cluster-image-registry-operator/pull/387

Reply:
Yes, we need to get #387 merged first.

Reply:
I have rebased this PR on top of #387 .